### PR TITLE
feat: add rtt information [ping]

### PR DESCRIPTION
**Pull request by rsommerard**

### CODE REVIEW

File: measures/src/main/java/io/apisense/network/ping/ICMPTask.java [hunk 1]
```diff
@@ -110,45 +118,70 @@ private ICMPResult launchPingCommand(String command) throws PINGException, IOExc
...
-        return new ICMPResult(startTimeMs, endTime, config, hostname, ip, latency, currentTtl);
       }
+
       matcher = PING_RESPONSE_SUCCESS_NO_HOSTNAME.matcher(line);
+
       if (matcher.matches()) {
         ip = matcher.group(1);
         latency = Float.valueOf(matcher.group(2)).longValue(); // milliseconds
-        return new ICMPResult(startTimeMs, endTime, config, null, ip, latency, currentTtl);
+        hostname = null;
```
Comment (aveuiller) [inline measures/src/main/java/io/apisense/network/ping/ICMPTask.java]: This match is exclusive with the previous one, so it's useless to set hostname back to null.

File: measures/src/main/java/io/apisense/network/ping/Rtt.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package io.apisense.network.ping;
+
+/**
+ * Definition of a ping RTT.
+ */
+public class Rtt {
+
+    private final float min;
```
Comment (aveuiller) [inline measures/src/main/java/io/apisense/network/ping/Rtt.java]: Create `getters`, or set the attributes as `public`

**Review by aveuiller — CHANGES_REQUESTED** [review]

File: measures/src/main/java/io/apisense/network/ping/Rtt.java (hunk 2 above)
Comment (rsommerard) [reply measures/src/main/java/io/apisense/network/ping/Rtt.java]: Immutability is better, the RTT is the result of the ping so there is no need to modify it, don't you?

File: measures/src/main/java/io/apisense/network/ping/ICMPTask.java (hunk 1 above)
Comment (rsommerard) [reply measures/src/main/java/io/apisense/network/ping/ICMPTask.java]: OK with that 👍

**Review by aveuiller — APPROVED** [review]

**Pull request merged**